lib/ex_webrtc/peer_connection/configuration.ex

```diff
@@ -60,10 +68,13 @@ defmodule ExWebRTC.PeerConnection.Configuration do
                                   {extmap.uri, extmap}
                                 end)
 
-  @mandatory_video_rtp_hdr_exts Map.new([:mid, :twcc], fn ext_shortcut ->
-                                  extmap = Map.fetch!(@rtp_hdr_extensions, ext_shortcut).ext
-                                  {extmap.uri, extmap}
-                                end)
+  @mandatory_video_rtp_hdr_exts Map.new(
+                                  [:mid, :twcc, :rtp_stream_id, :repaired_rtp_stream_id],
```

Comment:
It is ok, although not necessary. Otherwise, the user will need to add the extension themselves, which is very error-prone and hard to debug. I would leave it as it is at least until we tackle the codec/extension config in general.